wremnants/include/lowpu_utils.h

```diff
@@ -162,6 +162,68 @@ Vec_f trigFilter(Vec_f tag, Vec_f trigMatch) {
 
 
 
+Vec_i indices_(const Vec_f& vec, const int& start = 0) {
+    Vec_i res(vec.size(), 0);
+    std::iota(std::begin(res), std::end(res), start);
+    return res;
+}
+
+Vec_i indices_(const int& size, const int& start = 0) {
+    Vec_i res(size, 0);
+    std::iota(std::begin(res), std::end(res), start);
+    return res;
+}
+
+
+// for Z
+const double MZ_GEN_ = 91153.509740726733;
+const double GAMMAZ_GEN_ = 2493.2018986110700;
+const double MW_GEN_ = 80351.812293789408;
+const double GAMMAW_GEN_ = 2090.4310808144846;
+
+double computeBreitWignerWeight(double massVgen, double offset, int type) {

```

Comment:
I think this could reasonably go in the theoryTools.h file as it is of general use. We will probably want to check some things in the high pu using these weights as well.

Reply:
Moved to utils.h